docs/guides/core-concepts/test-isolation.mdx

```diff
@@ -42,7 +42,7 @@ all:
 - [stubs](/api/commands/stub)
 - [viewport changes](/api/commands/viewport)
 
-In additional to a clean test slate, Cypress also believes in running tests in a
+In addition to a clean test slate, Cypress also believes in running tests in a
 clean browser context such that the application or component under test behaves
 consistently when ran. This behavior is described as `testIsolation`.
```

Comment:
It looks like there is another error in this sentence:

```suggestion
consistently when run. This behavior is described as `testIsolation`.
```

(short for "when it is run").